staging/src/k8s.io/kube-scheduler/config/v1alpha1/types.go

```diff
@@ -84,6 +84,15 @@ type KubeSchedulerConfiguration struct {
 	// If this value is nil, the default value will be used.
 	BindTimeoutSeconds *int64 `json:"bindTimeoutSeconds"`
 
+	// PodInitialBackoffDurationSeconds is the initial backoff duration for unschedulable pods.
+	// It should be greater than 0. If this value is nil, the default value will be used.
+	PodInitialBackoffDurationSeconds *int64 `json:"podInitialBackoffDurationSeconds"`
```

Comment:
prefer `PodInitialBackoffSeconds` (much more consistent with other time-measuring fields in the API)?